content/gophergala/data-mining.md

```diff
@@ -0,0 +1,154 @@
++++
+author = ["Marcel Hauf"]
+date = "2015-02-02T17:10:37+01:00"
+linktitle = "data-mining"
```

Comment:
linktitle (Short Title) generally isn't hyphenated and may not even be needed in this case.